webapp/components/admin_console/email_authentication_settings.jsx

```diff
@@ -84,7 +84,7 @@ export default class EmailAuthenticationSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.email.allowUsernameSignInTitle'
-                            defaultMessage='Allow Sign In With Username: '
+                            defaultMessage='Enable login with username: '
```

Comment:
Propose we use "sign-in" to match the UI, rather than "login" which isn't used in the UI?